nbs/60_medical.imaging.ipynb

```diff
@@ -80,6 +80,56 @@
     "_all_ = ['DcmDataset', 'DcmTag', 'DcmMultiValue', 'dcmread', 'get_dicom_files']"
```

Comment:
<p>Always put a <code>;</code>  at the end of a matplotlib line to avoid printing extra stuff</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2732/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>got it on including <code>;</code>  to remove all extra stuff</p>